app/db/experiencia/profissional/crud.py

```diff
@@ -50,6 +50,7 @@ def create_experiencia(
             pessoa_id=pessoa_id,
         )
     except Exception as e:
+        print('CORRIGIR FUTURAMENTE. Exceção encontrada:', e)
         print(e)
```

Comment:
Remover essa linha pois a exceção já está sendo exibida na linha acima.

Reply:
sim eu esqueci de remover ja fiz isso e to enviando